examples/prescriptors/neat/train_prescriptor.py

```diff
@@ -11,16 +11,14 @@
 import numpy as np
 import pandas as pd
 
```

Comment:
same changes as in prescribe.py

Reply:
Maybe `prescribe.py` could point to this code

Reply:
there probably is a nice way to refactor parameterize the evaluation so that it can be used in both. i'll look into that for the weighting example